docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -993,8 +1135,8 @@ Note that this example of embedded property syntax is equivalent to the followin
 
 *Filter Example - Multiple Pivots*
 
-- From an initial set of ``it:exec:file:read`` operations, return only those operations where the base file name
-  of the PDB path of the file performing the read operation is ``moonclient2.pdb``:
+- From an initial set of ``it:exec:file:read`` operations, filter results to only those operations where
+  the base file name of the PDB path of the file performing the read operation is ``moonclient2.pdb``:
 
 .. storm-pre:: [ ( it:exec:file:read=1fe9e8690b76f84680468b4018cc3655 :sandbox:file=sha256:708840e33fe6632ba75fcf86564e0a2ece4f7b265390afe15aa5ae383a917e65 ) ( file:bytes=sha256:708840e33fe6632ba75fcf86564e0a2ece4f7b265390afe15aa5ae383a917e65 :mime:pe:pdbpath='e:/code/moon1.5/release/moonclient2.pdb' ) ] it:exec:file:read +:sandbox:file::mime:pe:pdbpath::base=moonclient2.pdb
```

Comment:
this also seems to do an extra lift of the `it:exec:file:read` node?

Reply:
NOTHING TO SEE HERE